pkg/kubeadm/kubeadm.go

```diff
@@ -162,6 +163,11 @@ func (k *Config) CreatePKI() (err error) {
 
 // CreateKubeConfig - Creates all the kubeconfig files requires for masters
 func (k *Config) CreateKubeConfig() (err error) {
+	if k.KubeletID == "" {
+		if k.KubeletID, err = os.Hostname(); err != nil {
```

Comment:
What would be the hostname? If kubelet runs in a container, does that mean that the hostname changes everytime kubelet restarts?